tests/render/renderer/test_notebook_renderer.py

```diff
@@ -1495,3 +1500,76 @@ def test_complex_suite(warning_suite):
     for obs_cell, expected_cell in zip(obs["cells"], expected["cells"]):
         assert obs_cell == expected_cell
     assert obs == expected
+
+
+def test_notebook_execution_with_pandas_backend(titanic_data_context):
+    """
+    To set this test up we:
+
+    - create a suite
+    - add a few expectations (both table and column level)
+    - verify that no validations have happened
+    - create the suite edit notebook by hijacking the private cli method
+
+
+    We then:
+    - execute that notebook (Note this will raise various errors like
+    CellExecutionError if any cell in the notebook fails
+    - create a new context from disk
+    - a validation has been run with our expectation suite
```

Comment:
```suggestion
    - verify that a validation has been run with our expectation suite
```
Small word change, and maybe a chance here to add the additional tests suggested below

Reply:
Good catch. Done